service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadDocumentsController.java

```diff
@@ -12,19 +12,30 @@
 import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 import uk.gov.hmcts.reform.fpl.service.DocumentsValidatorService;
+import uk.gov.hmcts.reform.fpl.service.UploadDocumentsService;
+
+import java.util.List;
 
 @Api
 @RestController
 @RequestMapping("/callback/upload-documents")
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class UploadDocumentsController extends CallbackController {
     private final DocumentsValidatorService documentsValidatorService;
+    private final UploadDocumentsService uploadDocumentsService;
 
     @PostMapping("/mid-event")
     public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestBody CallbackRequest callbackrequest) {
+        CaseData caseDataBefore = getCaseDataBefore(callbackrequest);
```

Comment:
the scope of this variable could be reduced to the if statement

Reply:
agree.